x-pack/plugins/remote_clusters/public/sections/components/remote_cluster_form/validators/validate_seeds.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+export function validateSeeds(seeds, seedInput) {
+  const seedsHaveBeenCreated = seeds.some(seed => Boolean(seed.trim()));
+
+  if (seedsHaveBeenCreated) {
+    return undefined;
```

Comment:
IMO it makes more sense that a validator function return either an error or `null`.

```js
const error = validateSeeds(<...args>); // null
```

`undefined` seems like it hasn't been validated.